enterprise/security/src/main/java/org/neo4j/server/security/enterprise/auth/LdapRealm.java

```diff
@@ -224,6 +337,12 @@ private void configureRealm( Config config )
                 parseGroupToRoleMapping( config.get( SecuritySettings.ldap_authorization_group_to_role_mapping ) );
     }
 
+    private String parseLdapServerUrl( String rawLdapServer )
+    {
+        return (rawLdapServer == null) ? null :
```

Comment:
Hmm, do we need to make null-checks on config values? Should we be doing that in more places, or if there a special reason for this setting?